client/src/components/CustomFields.js

```diff
@@ -286,9 +286,42 @@ const EnumField = fieldProps => {
 
 const enumHumanValue = (choices, fieldVal) => {
```

Comment:
Not part of the change, maybe good to directly update:
```js
const enumHumanValue = () => {}
function enumHumanValue() {} // Shorter and less operators
```
Other motivation: some say that it as adventages in debugging as the name of the function stays preserved.